Node Status Monitor/NodeJS/node_monitor.js

```diff
@@ -0,0 +1,197 @@
+/* imports*/
```

Comment:
For consistency, use only `=>` for array processing.

Reply:
Done!